index.bs

```diff
@@ -6064,6 +6076,205 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
+
+If the [=client device=] is incapable of generating a [=hardware-bound device key pair=], or the registration or authentication operation fails for any reason, this extension is ignored and no [=hardware-bound device key pair=] is created. In this case, there is no [=devicePubKey=] extension output generated.
+
+The [=hardware-bound device key pair=] is not on its own a [=user credential=] and does not have its own [=credential ID=]. Instead, the returned [=device public key=] is a device-specific contextual attribute of its associated [=user credential=]. That is, when that [=user credential=] is used&mdash;along with the [=devicePubKey=] extension&mdash;on a particular [=client device=], a particular [=device public key=] is returned by the extension.
+
+Issue(1665): TODO: explicitly add the notion of "synced user credentials" to the spec.
+
+
+### Relying Party Usage ### {#sctn-device-publickey-extension-usage}
+
+This extension is intended for use by those [=[RPS]=] employing risk-analysis systems informing their sign-in decisions. This extension provides a "device continuity" signal when used consistently with both {{CredentialsContainer/create()|navigator.credentials.create()}} and {{CredentialsContainer/get()|navigator.credentials.get()}} operations:
+
+A new [=device public key=] is returned to the [=[RP]=] (along with a new [=credential=]) when the `devicePubKey` extension is used with a {{CredentialsContainer/create()|navigator.credentials.create()}} call. The same [=device public key=] is returned to the [=[RP]=] along with [=assertions=] generated using the latter credential&mdash;i.e., via {{CredentialsContainer/get()|navigator.credentials.get()}} calls&mdash;if those {{CredentialsContainer/get()|get()}} calls occur on the <i>same device</i> as the {{CredentialsContainer/create()|create()}} call that created the credential.
+
+However, when a user's credential is synced to a different device, the [=[RP]=] will receive a new [=device public key=] on the initial {{CredentialsContainer/get()|navigator.credentials.get()}} invocation on the other device. Note that a synced credential will be exercised for authentication on a different device without a {{CredentialsContainer/create()|navigator.credentials.create()}} having been performed on that new device.
+
+A usage example is thus:
+
+>    Say that a sign-in request appears at a website along with some geolocation signal that has not been seen for this [=user account=] before, and is outside of the typical usage hours observed for the account. The risk may be deemed high enough not to allow the request, even with a synced credential. But if a signature by a [=device-bound  key=] that is well established for this user can also be presented, then that may tip the balance.
+
```

Comment:
I think we should avoid the words "sync" and "synced" as that is just one option for getting a credential on another device. We should be more generic.

> However, when a user's credential is **synced** to a different device,

"copied or moved"

>The risk may be deemed high enough not to allow the request, even with a **synced** credential.

"multi-device credential"

Reply:
done in 683ad4d